R-packages/evalcast/R/get_covidhub_predictions.R

```diff
@@ -1,3 +1,169 @@
+#' Get predictions from forecasters on the COVID Hub
+#'
+#' Simply converts the predictions of forecasters submitting to the [COVID
+#' Hub](https://github.com/reichlab/covid19-forecast-hub/) to the format of a
+#' predictions card, so it can be easily evaluated and compared.
+#'
+#' @param covidhub_forecaster_name A vector of strings indicating the
+#'   forecasters (matching what it is called on the COVID Hub).
+#' @param forecast_dates Vector of Date objects (or strings of the form
+#'   "YYYY-MM-DD") indicating dates on which forecasts will be made. If `NULL`,
+#'   the default, then all currently available forecast dates from the given
+#'   forecaster in the COVID Hub will be used.
+#' @param geo_values vector of character strings containing FIPS codes of
+#'   counties, or lower case state abbreviations (or "us" for national). The
+#'   default "*" fetches all available locations
+#' @param forecast_type "quantile", "point" or both (the default)
+#' @param ahead number of periods ahead for which the forecast is required.
+#'   NULL will fetch all available aheads
+#' @param incidence_period one of "epiweek" or "day". NULL will attempt to
+#'   return both
+#' @param signal this function supports only "confirmed_incidence_num",
+#'   "deaths_incidence_num", and/or "deaths_cumulative_num" (those currently
+#'   forecast by the COVIDhub-ensemble). For other types, use one of the
+#'   alternatives mentioned above
+#' @param predictions_cards An object of class `predicitions_cards` that
+#'   contains previously retrieved predictions. If provided, files will not be
+#'   retrieved from Reichlab for any forecaster / forecast_date combos that are
+#'   present in the file.
+#' @param predictions_cards_filename .RDS file containing existing predictions.
```

Comment:
I question the utility of this argument since it can be superseded by the previous argument in a single line, i.e.,
```
get_covidhub_predictions(names, predictions_cards_filename = "file.RDS")
```
vs.
```
get_covidhub_predictions(names, predictions_cards = readRDS("file.RDS"))
```

Reply:
I agree with @sgsmob here. Maybe just point out in the documentation that `predictions_cards = readRDS("file.RDS")` is a possible use case.

Reply:
Agreed, it doesn't need to be separate. Didn't change the predictions_cards documentation, as there's a number of formats somebody could choose to save data in.